### ISSUE TITLE
caddy2-filter is incompatible with caddy v2.6.3

### ISSUE THREAD

**Issue by W0n9** [sjtug/caddy2-filter#11]
When I tried to build with caddy v2.6.3, it raised an panic.
```bash
2023/02/09 08:50:50 [INFO] exec (timeout=0s): /home/w0n9/golang/go/bin/go get -d -v github.com/sjtug/caddy2-filter github.com/caddyserver/caddy/v2@v2.6.3 
go: downloading github.com/sjtug/caddy2-filter v0.0.0-20220728031114-2540c7e290e0
panic: internal error: can't find reason for requirement on github.com/google/pprof@v0.0.0-20210407192527-94a9f03dee38

goroutine 1 [running]:
cmd/go/internal/modget.(*resolver).updateBuildList.func1({{0xc000a1af90?, 0xc001328420?}, {0xc0001ac6f0?, 0xc000f35470?}})
        /usr/local/go/src/cmd/go/internal/modget/get.go:1760 +0x114
cmd/go/internal/modget.(*resolver).updateBuildList(0xc0003ce000, {0xb2d350, 0xc000188000}, {0x0, 0x0, 0x0})
        /usr/local/go/src/cmd/go/internal/modget/get.go:1765 +0x597
cmd/go/internal/modget.(*resolver).applyUpgrades(0xc0003ce000, {0xb2d350, 0xc000188000}, {0x0?, 0x2?, 0x472445?})
        /usr/local/go/src/cmd/go/internal/modget/get.go:1312 +0x105
cmd/go/internal/modget.runGet({0xb2d350, 0xc000188000}, 0xc0001b8450?, {0xc00019e160, 0x2, 0x2})
        /usr/local/go/src/cmd/go/internal/modget/get.go:351 +0x45e
main.invoke(0xe27fa0, {0xc00019e130, 0x5, 0x5})
        /usr/local/go/src/cmd/go/main.go:225 +0x34e
main.main()
        /usr/local/go/src/cmd/go/main.go:179 +0x7d1
2023/02/09 08:50:53 [FATAL] exit status 2
```
## How to reproduce this error
```bash
xcaddy build v2.6.3 --with github.com/sjtug/caddy2-filter
```

**Comment by jasonalsing** [comment]
Same of course on 2.6.4, looking to upgrade hopefully this can be fixed soon.

**Comment by nrosier** [comment]
Created PR #12 to fix build with v2.6.4
Succesfully compiled against my fork
